src/components/specs/Container.spec.js

```diff
@@ -0,0 +1,20 @@
+import { create } from 'react-test-renderer';
+import Container from '../Container';
+
+jest.mock('react-router-dom', () => ({
+  Link: 'Link',
+  Route: 'Route',
+  Switch: 'Switch',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/megwali/spacex/pullRequest?prid=7693722)
```suggestion
  Switch: "Switch",
```